tests/unit/operations_v1/test_operations_rest_client.py

```diff
@@ -82,16 +87,68 @@
 }
 
 
+if GOOGLE_AUTH_AIO_INSTALLED:
```

Comment:
`TEST_TRANSPORT_CREDS_PARAMS` can be renamed to `TRANSPORT_WITH_CREDENTIALS` and moved in here.

Reply:
This is removed and another parameter is used instead.